src/formatters/vsoFormatter.ts

```diff
@@ -27,7 +27,7 @@ export class Formatter extends AbstractFormatter {
         formatterName: "vso",
         description: "Formats output as VSO/TFS logging commands.",
         descriptionDetails: Utils.dedent`
-            Integrates with Visual Studio Online and Team Foundation Server by outputting errors
+            Integrates with Azure DevOps(previously known as Visual Studio Online and Team Foundation Server) by outputting errors
```

Comment:
> `Visual Studio Online and Team Foundation Server`

While you're here, would you mind changing that to `Visual Studio Online, Team Foundation Server, or Visual Studio Team Services`?

Reply:
`Integrates with Azure DevOps (previously known as Visual Studio Online, Team Foundation Server, or Visual Studio Team Services) by outputting errors`

@JoshuaKGoldberg Does the above change look good? Once you confirm, I will proceed to commit

Reply:
Looks perfect, thanks!

Reply:
Thanks, changes are committed